skimage/measure/_regionprops.py

```diff
@@ -640,6 +642,9 @@ def regionprops(label_image, intensity_image=None, cache=True):
         Determine whether to cache calculated properties. The computation is
         much faster for cached properties, whereas the memory consumption
         increases.
+    coordinates : DEPRECATED
+        This argument is deprecated and will be removed in a future version
+        of scikit-image.
```

Comment:
Maybe remove this from the docstring? Is that current library practice? The warning message gives all the information that is needed in my opinion and is likely the way users will notice this depreciation anyway.

Reply:
We've been trying to use markers like `.. deprecated :` to make it more explicit and to help people avoid warnings before their code is run
https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html?highlight=deprecated#directive-deprecated

Reply:
In that case isn't that directive missing here? Not that the directive seems to have any special styling in the current theme but that may change in the future.

Reply:
Added the directive, thanks for the info!